client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -303,13 +434,23 @@ export const getFormErrors = createSelector(
 		const { countriesData } = shippingLabel.storeOptions;
 		const { form, paperSize } = shippingLabel;
 		if ( isEmpty( form ) ) {
-			return;
+			return {};
```

Comment:
Why is a return value of `{}` needed?

Reply:
I don't know if it was crashing in any particular way, but it's good practice to have predictable return values. As it was before this change, this function could return an object, an empty object, or `undefined`. That's asking for trouble and the solution is literally 3 characters :)